inspirehep/config.py

```diff
@@ -52,10 +52,11 @@
 #   some ORCIDs -> "^(0000-0002-7638-5686|0000-0002-7638-5687)$"
 FEATURE_FLAG_ORCID_PUSH_WHITELIST_REGEX = '.*'
 FEATURE_FLAG_ENABLE_FUZZY_MATCHER = False
```

Comment:
could you please also enable ``FEATURE_FLAG_ENABLE_FUZZY_MATCHER``?

Reply:
:+1: done